imports/ftp_ecmwf_download.py

```diff
@@ -186,12 +186,13 @@ def remove_old_ftp_downloads(folder):
     date_now = datetime.datetime.utcnow()
     all_paths = glob(os.path.join(folder,'Runoff*netcdf*'))
     for path in all_paths:
-        date_file = datetime.datetime.strptime(os.path.basename(path).split('.')[1],'%Y%m%d')
-        if date_now - date_file > datetime.timedelta(1):
-            if os.path.isdir(path):
-                rmtree(path)
-            else:
-                os.remove(path)
+	date_file = datetime.datetime.strptime(os.path.basename(path).split('.')[1],'%Y%m%d')
+        if os.path.isdir(path):
+            rmtree(path)
+        else:
+            os.remove(path)
+	if date_now - date_file < datetime.timedelta(1):
+	    os.mkdir(path)

```

Comment:
The idea is to delete the stuff in the folder (the forecast) but keep the folder so it will know where to get the next download.